wowchemy/archetypes/authors/_index.md

```diff
@@ -6,6 +6,9 @@ title: "{{ replace .Name "-" " " | title }}"
 authors:
 - "{{ urlize .Name }}"
 
+aliases:
+- "{{ .Name }}"
```

Comment:
Why do we appear to be duplicating the author display name (`title` field) as an alias?

Reply:
https://github.com/wowchemy/wowchemy-hugo-modules/pull/1805/files?file-filters%5B%5D=.html#diff-8f20a3fea090d788a0c219ae877d58c6R28

`$lookup` strictly considers the names on disk. Take for example... your site has `authors/gcushen`, a few years down the line you change your online names, e.g. to `geo`. If your site has used `gcushen` as the author anywhere, but the folder name on disk is now `geo` – any pages with `authors: [ gcushen, ]` won't resolve, unless you manually add that as an alias.

It may require running a `urlize` on the `.Name`, but it's effectively to protect against folder name changes and pre-seed an example of what `aliases` (err... `nicknames`) should contain.